package.json

```diff
@@ -264,6 +263,7 @@
     "resin-cli-form": "^2.0.2",
     "resin-cli-visuals": "^1.8.0",
     "resin-compose-parse": "^2.1.3",
+    "resin-discoverable-services": "^2.0.3",
```

Comment:
`balena-sync` v11.0.2, which this PR proposes to replace, was using `resin-discoverable-services` "version" `git+https://github.com/resin-io-modules/resin-discoverable-services.git#find-on-all-interfaces` rather than version `2.0.3`. The diff of `npm-shrinkwrap.json` confirms this. I gather that there are logic differences between `resin-discoverable-services` v2.0.3 and branch `resin-discoverable-services.git#find-on-all-interfaces`. Here's the diff: https://github.com/balena-io-modules/resin-discoverable-services/compare/v2.0.3...find-on-all-interfaces

The commit of branch `find-on-all-interfaces` has this description:

```
Specifically listen on any valid IPv4 interface, to ensure we get responses from all NICs back.

* Filters responses so there are no duplicates.
* Uses modified `bonjour`->`multicast-dns` branches (experimental) to perform multi-nic listen.

WARNING: This should not be merged onto `master`, as more work needs to be done to the branches.
```

Despite that warning message in the commit description, the balena CLI has been using that branch in the `balena scan` command for more than 4 years (it predates me!), and to a lesser extent in the `balena join` and `balena leave` commands.

I can't remember if this was discussed as part of PR balena-io-modules/balena-sync#167 - I've now added a similar comment over there: https://github.com/balena-io-modules/balena-sync/pull/167/files#r803160932

The easy way out would be to maintain the use of `git+https://github.com/resin-io-modules/resin-discoverable-services.git#find-on-all-interfaces` in the balena CLI's `package.json`, while the "clean" "proper" solution would be to create a PR in `resin-discoverable-services` and merge that branch to master.